tests/integration/test_types_integration.py

```diff
@@ -300,114 +306,114 @@ def query_time_with_timezone(trino_connection, tz_str: str):
         # min supported time(3)
         .add_field(
             sql=f"TIME '00:00:00 {tz_str}'",
-            python=time(0, 0, 0).replace(tzinfo=tz))
+            python=time(0, 0, 0, tzinfo=tz))
```

Comment:
Clearer/cleaner to specify the time zone when creating the object.

Reply:
are there any other occurrences of this in production code as well?